queries.sql

```diff
@@ -15,3 +15,34 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
 -- Query 8: Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+/*Vet clinic database: query and update animals table*/
+-- Query 1: update the animals table by setting the species column to unspecified
+UPDATE animals SET species = 'unspecified';
+-- Query 2: Update the animals table by setting the species column to digimon for all animals that have a name ending in mon
```

Comment:
* As required in the [project requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction%20update%20the%20animals%20table%20by%20setting%20the%20species%20column%20to%20unspecified.%20Verify%20that%20change%20was%20made.%20Then%20roll%20back%20the%20change%20and%20verify%20that%20the%20species%20columns%20went%20back%20to%20the%20state%20before%20the%20transaction.) you are to 

> update the animals table by setting the species column to unspecified

> Verify that the change was made on the animal table 

> Then roll back the change and verify that the species columns went back to the state before the transaction. Kindly work on all these 🙏